skt/runner.py

```diff
@@ -148,6 +155,12 @@ def getconsolelog(self, jobid=None):
         return url
 
     def _forget_cid(self, cid):
+        """
+        Remove a job or recipe from the current list
+
+        Args:
+            cid: The job (J:xxxxx) or recipe (R:xxxxx) id
```

Comment:
This will have to do for now, but I think we should use Beaker's [official term](https://www.mankier.com/1/bkr#Description-Specifying_tasks) "taskspec" instead? E.g. "cid: The job or recipe taskspec (universal ID)".